packages/picasso/src/Container/Container.tsx

```diff
@@ -142,6 +142,8 @@ export const Container = documentable(
               [classes.flex]: flex,
               [classes.inline]: inline,
               [classes.column]: direction === 'column',
+              [classes.rowReverse]: direction === 'row-reverse',
+              [classes.columnReverse]: direction === 'column-reverse',
```

Comment:
I would prefer following the same approach as others for the same property `direction`
```suggestion
              [classes[kebabToCamelCase(direction || '')]]: direction,
```

Reply:
@ozgurkececioglu 
I updated code the way you suggested